prow/cmd/checkconfig/main.go

```diff
@@ -459,7 +459,8 @@ func strictBranchesConfig(c config.ProwConfig) (*orgRepoConfig, error) {
 				for branchName := range repo.Branches {
 					branch, err := repo.GetBranch(branchName)
 					if err != nil {
-						return nil, err
+						return nil, fmt.Errorf("error for branch %s/%s/%s: %w",
```

Comment:
it's probably a bit bikesheddy, but I think that "branch org/repo/branch" is still a bit confusing if you imagine it in an output. How about "error for branch 'branch' of 'org/repo:'" or maybe "error for repo=org/repo branch=branch: err"?

Reply:
Updated.